package.json

```diff
@@ -62,19 +62,19 @@
         "@graphql-codegen/cli": "^2.6.2",
         "@graphql-codegen/typescript": "^2.4.5",
         "@graphql-codegen/typescript-operations": "^2.3.2",
-        "@storybook/addon-actions": "^6.4.9",
-        "@storybook/addon-essentials": "^6.4.9",
-        "@storybook/addon-links": "^6.4.9",
-        "@storybook/builder-webpack5": "^6.4.9",
-        "@storybook/manager-webpack5": "^6.4.9",
-        "@storybook/react": "^6.4.9",
+        "@storybook/addon-actions": "^6.5.15",
+        "@storybook/addon-essentials": "^6.5.15",
+        "@storybook/addon-links": "^6.5.15",
+        "@storybook/builder-webpack5": "^6.5.15",
+        "@storybook/manager-webpack5": "^6.5.15",
+        "@storybook/react": "^6.5.15",
```

Comment:
Could you please check are we using storybook? I remember that someone remove folder with storybook one day. Maybe me 😅 
In this case we can remove 6 dependencies here as well

Reply:
surly :smile: will check them later